### PR TITLE
Set plugin in the instrument.sh

### PR DESCRIPTION
**Pull request by dszmigielski**

### CODE REVIEW

File: instrument.sh [hunk 1]
```diff
@@ -141,4 +141,5 @@ if [ "$ENABLE_PROFILING" = "true" ]; then
 
   # Configure the bytecode instrumentation configuration file
   export OTEL_DOTNET_AUTO_INTEGRATIONS_FILE="$OTEL_DOTNET_AUTO_HOME/integrations.json"
+  export OTEL_DOTNET_AUTO_PLUGINS="Splunk.OpenTelemetry.AutoInstrumentation.Plugin, Splunk.OpenTelemetry.AutoInstrumentation, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null"
```
Comment (pellared) [inline instrument.sh:144]: @Kielek I see that MinVer works only for NuGet packages. Shouldn't we use

```
  <PropertyGroup>
    <Version>PUT.VERSION.HERE</Version>
  </PropertyGroup>
```

instead?

Comment (pellared) [reply instrument.sh:144]: @pjanotti @RassK any thoughts?

**Review by pellared — APPROVED** [review]

**Comment by pellared** [comment]
What about `Splunk.OTel.DotNet.psm1`?

File: instrument.sh (hunk 1 above)
Comment (dszmigielski) [reply instrument.sh:144]: Easier only in a sense that we would not have to change version in 2 places with every release, imo we should do that anyway.

**Comment by pellared** [comment]
`ci / shell-scripts` are failing becasue it tries to use the script on the "latest release". We should test the scripts on the "build output" and have a seperate workflow to test the scripts after the release... The same problem is in upstream.

File: instrument.sh (hunk 1 above)
Comment (Kielek) [reply instrument.sh:144]: MinVer is working as expected also for libraries.
FileVersion != Assembly version. See https://github.com/adamralph/minver#version-numbers and https://learn.microsoft.com/en-ca/dotnet/standard/library-guidance/versioning#version-numbers

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -30,13 +30,15 @@ jobs:
       - run: ./build.cmd Workflow
       - run: brew install coreutils
         if: ${{ runner.os == 'macOS' }}
+      - name: Create test directory
+        run: sudo mkdir -p ${{ matrix.log-dir }} && sudo chmod a+rwx ${{ matrix.log-dir }}  # workaround before next release
+        if: ${{ runner.os != 'Windows' }}
```
Comment (pellared) [inline .github/workflows/ci.yml:35]: This should not be needed anymore

**Comment by RassK** [comment]
> What about `Splunk.OTel.DotNet.psm1`?

I'll take it, it has more issues than just the missing plugin config.

File: instrument.sh (hunk 1 above)
Comment (pellared) [reply instrument.sh:144]: I think that @RassK said somwhere that this line could be simplified to:


```suggestion
  export OTEL_DOTNET_AUTO_PLUGINS="Splunk.OpenTelemetry.AutoInstrumentation.Plugin, Splunk.OpenTelemetry.AutoInstrumentation"
```

**Review by Kielek — APPROVED** [review]

**Pull request merged**